tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_interval.py

```diff
@@ -46,10 +99,33 @@ def get_configuration(request):
 
 
 def test_interval(get_configuration, configure_environment, restart_modulesd):
-    """
-    Check if modulesd waits `interval` between one vulnerability detector scan and another.
-    """
-
+    '''
+    description: Check if modulesd waits `interval` between one vulnerability detector scan and another. To do this, it
+                 checks in the `ossec.log` file appears the corresponding message.
+
+    wazuh_min_version: 4.2.0
+
+    parameters:
+        - get_configuration:
+            type: fixture
+            brief: Get configurations from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing.
+        - restart_modulesd:
+            type: callable
+            brief: Restart the `wazuh-modulesd` daemon.
+
+    assertions:
+        - Verify that the Vulnerability Detector process thread sleeps the time set, checking `ossec.log` message.
+
+    input_description:
+        - Test cases are defined in the list interval_values and interval_units. This test gets their configuration of
+          the wazuh_interval.yaml file.
+
+    expected_output:
+        - 'Missing sleep between scans'
```

Comment:
```suggestion
        - r'.* Sleeping for (.*)...'
```

Reply:
Done in this [link](https://github.com/wazuh/wazuh-qa/commit/54d592fab538ac1cb516607b3d9c1eea2e5d3c65)